server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js

```diff
@@ -9,6 +9,17 @@ const { Manuscript, User } = require('@elifesciences/xpub-model')
 const { File } = require('@elifesciences/xpub-model')
 const { S3Storage } = require('@elifesciences/xpub-controller')
 
+async function debugMsFiles(id, user) {
```

Comment:
`ms` can easily be confused with milliseconds in the context of manuscripts, given the recent timeout problems

Reply:
also the captain of the Titanic used to say that

Reply:
That's why the PR has :sunk: